src/sentry/tasks/store.py

```diff
@@ -280,7 +280,7 @@ def save_event(cache_key=None, data=None, start_time=None, event_id=None, **kwar
     except HashDiscarded as exc:
         info_logger.info(
```

Comment:
While you're in here, can you change this to `debug`? This is going to harm logging infra as it gets popular.